doctr/models/_utils.py

```diff
@@ -150,7 +150,11 @@ def estimate_orientation(img: np.ndarray, n_ct: int = 50, ratio_threshold_for_li
             angles.append(angle)
         elif w / h < 1 / ratio_threshold_for_lines:  # if lines are vertical, substract 90 degree
             angles.append(angle - 90)
-    return -median_low(angles)
+
+    if len(angles) == 0:
+        return 0  # in case no angles is found
```

Comment:
Mind adding a unittest case for this please?

Reply:
ok done!